phys/module_ra_rrtmg_sw.F

```diff
@@ -10356,10 +10355,20 @@ SUBROUTINE RRTMG_SWRAD(                                        &
 
 !-----CALCULATE SHORT WAVE RADIATION
 !                                                              
-#ifdef CLWRFGHG
+IF(xtime.eq.0 .or. (mod(xtime,1440.).eq.0)) THEN
+ IF(ghg_input.eq.1) then
```

Comment:
@smileMchen Same change suggested above could be applied here. But no need to print these values twice. LW is called first, so let it print.